wren-ai-service/tests/pytest/services/test_ask.py

```diff
@@ -76,6 +72,14 @@ def indexing_service():
     )
 
 
+@pytest.fixture
+def service_metadata():
+    return {
+        "models_metadata": {},
+        "service_version": "",
+    }
```

Comment:
nitpick: it seems the same as the default which the function `trace_metadata` can't got from kwargs, I think giving a mock value is better to identify for testing and the default